test/test_matplotlib.py

```diff
@@ -28,3 +28,29 @@ def test_pdf(selenium):
     selenium.run("import io")
     selenium.run("fd = io.BytesIO()")
     selenium.run("plt.savefig(fd, format='pdf')")
+
+
+def test_rendering(selenium):
+    selenium.load_package("matplotlib")
+    selenium.run("from matplotlib import pyplot as plt")
+    selenium.run("import numpy as np")
+    selenium.run("import pyodide, io, base64")
+    selenium.run("data = io.BytesIO()")
+    selenium.run("t = np.arange(0.0, 2.0, 0.01)")
+    selenium.run("s = 1 + np.sin(2 * np.pi * t)")
+    selenium.run("plt.plot(t, s, linewidth=1.0, marker=11)")
+    selenium.run("plt.plot(t, t)")
+    selenium.run("plt.grid(True)")
+    selenium.run("plt.show()")
+    # png_button = selenium.driver.find_element_by_xpath("//button[contains(text(), 'png')]")
+    # png_button.click()
+    # selenium.run("canvas_data = plt.gcf().canvas.get_pixel_data()")
+    # selenium.run("ref_data = pyodide.open_url('test/canvas.png').getvalue()")
+    # ref_data = selenium.run("io.open(ref_data, encoding='rb', buffering = 0)")
+    # assert ref_data == 1
+    selenium.run("plt.gcf().canvas.print_png(data)")
+    y = selenium.run("base64.b64encode(data.getvalue()).decode('utf-8')")
+    selenium.run("ref_data = pyodide.open_url('test/canvas.png').getvalue()")
+    z = selenium.run("base64.b64encode(io.BytesIO(ref_data.encode('utf-8')).getvalue()).decode('utf-8')")
+    # x = selenium.run("base64.b64encode(data.getvalue()).decode('ascii') == ref_data")
+    assert y == z
```

Comment:
I doubt this will work to send the numpy arrays over the wire like this.  I would just compare them in Python code and return the boolean result of `np.all(y == z)` over to the pytest side.

Reply:
Yes, that makes sense. Sorry about that.